pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -626,6 +646,24 @@ func (o *MustGatherOptions) newPod(node, image string) *corev1.Pod {
 					ImagePullPolicy: corev1.PullIfNotPresent,
 					// always force disk flush to ensure that all data gathered is accessible in the copy container
 					Command: []string{"/bin/bash", "-c", "/usr/bin/gather; sync"},
+					Env: []corev1.EnvVar{
+						{
+							Name: "NODE_NAME",
+							ValueFrom: &corev1.EnvVarSource{
+								FieldRef: &corev1.ObjectFieldSelector{
+									FieldPath: "spec.nodeName",
+								},
+							},
+						},
+						{
+							Name: "POD_NAME",
+							ValueFrom: &corev1.EnvVarSource{
+								FieldRef: &corev1.ObjectFieldSelector{
+									FieldPath: "metadata.name",
+								},
+							},
+						},
+					},
```

Comment:
What are these used for?

Reply:
As mentioned in the commit message:

> This could be useful for users which specify `--node-selector` and capture must-gather data across multiple nodes, allowing them to separate the captured and downloaded data between nodes.

i.e: to have a separator for all downloaded files on a per node / pod basis.